styles.css

```diff
@@ -73,6 +72,63 @@ body {
   to { color: darkslateblue; }
 }
 
+.nav-item {
+  display: none;
+}
+
+@media screen and (max-width: 768px) {
+  .nav-item {
+    visibility: hidden;
+    opacity: 0;
+    width: 100%;
+    height: 120vh;
+    background: #6070ff;
+    display: flex;
+    position: fixed;
+    top: 0;
+    left: 100%;
+    transition: left 0.8s, visibility 1.8s, opacity 0.4s;
+    padding: 8rem 2rem;
+    z-index: 50;
+    flex-direction: column;
+    align-items: flex-start;
+    mix-blend-mode: multiply;
+    margin-top: -6rem;
+    backdrop-filter: blur(8px);
+  }
+
+  .nav-item li {
+    margin-bottom: 40px;
+  }
+
+  .nav-item li a {
+    font-size: 32px;
+    font-weight: 600;
+    line-height: 44px;
+    color: #fff;
+    transition: color 1s;
+  }
+
+  .nav-item li a:hover {
+    color: #000;
+  }
```

Comment:
- [ ] [Optional][Highly Recommended] Please add the correct margin values for the mobile menu options and try to stick to the given design as much as possible.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/223501308-ad34dca3-8258-4794-9ccb-31a2dde1316d.png)  |![image](https://user-images.githubusercontent.com/68796232/223501282-8d203a82-8fb2-4a5b-a0a2-9b88fd43a448.png)  |

### ⚠️I marked this as optional because the last reviewer forgot to mention it so please consider doing it⚠️